src/components/timeline/VerticalIndicators.js

```diff
@@ -88,18 +88,16 @@ export class VerticalIndicators extends React.PureComponent<Props> {
         innerWindowIDToPageMap &&
         data &&
         data.type === 'tracing' &&
-        data.category === 'Navigation'
+        data.category === 'Navigation' &&
+        data.innerWindowID
       ) {
-        const innerWindowID = data.innerWindowID;
```

Comment:
Flow was failing with `innerWindowID is missing in `ChromeDurationTraceEventPayload` and this is the only way I could silence it.